docs/TheArtOfHttpScripting.md

```diff
@@ -156,7 +156,7 @@
     curl https://curl.se
 
  you get a web page returned in your terminal window. The entire HTML document
- that that URL holds.
+ that URL holds.
```

Comment:
This was intentional. Just removing one makes it strange language. The second one could possibly be made a `the` to avoid the repitition.

Reply:
I also like "The entire HTML document that that URL holds" better